src/WasmDis.ts

```diff
@@ -1096,20 +1099,29 @@ class NameSectionNameResolver extends DefaultNameResolver {
       return `$${UNKNOWN_FUNCTION_PREFIX}${index}`;
     return isRef ? `$${name}` : `$${name} (;${index};)`;
   }
+
+  getVariableName(funcIndex: number, index: number, isRef: boolean): string {
+    const name = this._localNames[funcIndex] && this._localNames[funcIndex][index];
+    if (!name)
+      return `$${UNKNOWN_FUNCTION_PREFIX}${index}`;
```

Comment:
let's use DefaultNameResolver's method if name is not specified: `super.getVariableName(funcIndex, index, isRef)` ?

Reply:
Done.